esphome/components/light/effects.py

```diff
@@ -179,10 +180,12 @@ def addressable_color_wipe_effect_to_code(config, effect_id):
 
 @register_effect('addressable_scan', AddressableScanEffect, "Scan", {
     cv.Optional(CONF_MOVE_INTERVAL, default='0.1s'): cv.positive_time_period_milliseconds,
+    cv.Optional(CONF_SCAN_LENGTH, default=1): cv.All(cv.uint32_t, cv.Range(min=1))
```

Comment:
```suggestion
    cv.Optional(CONF_SCAN_LENGTH, default=1): cv.int_range(min=1),
```

Can be simplified - I don't think a user will ever exceed 2^32 leds :)
Also add `,` at the end for consistency.

Reply:
Done.